src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -126,6 +126,10 @@ class DocBaseViewer extends BaseViewer {
         this.docEl.classList.add('bp-doc');
         this.docEl.tabIndex = '0';
 
+        if (this.options.enableAnnotationsDiscoverability) {
+            this.docEl.classList.add('ba-discoverability-enabled');
```

Comment:
Do we need to clean it up in `destroy`?

Reply:
I'm not sure that we do, since `destroy()` will do `this.containerEl.innerHTML = '';`